media-gfx/hydrus/metadata.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person" proxied="yes">
+		<email>vaartis@kotobank.ch</email>
+		<name>Ekaterina Vaartis</name>
+	</maintainer>
+	<maintainer type="project" proxied="proxy">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription>
+		The hydrus network client is an application written for those who have large image/swf/webm collections. It browses
+		with tags instead of folders, a little like a booru on your desktop. Advanced users can share tags and files
+		anonymously through custom servers that any user may run
+	</longdescription>
+	<use>
+		<flag name="charts">Support for bandwidth charts</flag>
```

Comment:
We generally don't make USE flags for runtime-only deps, but pure Python is a grey area if it's really useful.

Reply:
Well, it's pure python so there's probably not much harm in keeping them.. the rebuild takes about 5-10 seconds. I can drop `miniupnpc` in favor of an optfeature, since it's sort of a niche feature and it's an external program, but i'd probably leave the rest in.